lib/delete-users-using-local-file.rb

```diff
@@ -0,0 +1,109 @@
+require 'zendesk_api'
+require 'rest-client'
+require 'json'
+require 'date'
+
+require_relative 'zendesk-setup.rb'
+
+# Setup vars
+
+diag = "true"
+url = "#{ENV['ZENDESK_URL']}/deleted_users/"
+lastyear = Date.today.next_day - 365
+
+File.open "data/soft_deleted_users.json" do |input_file|
+
+  file_data = JSON.load input_file
+
+  file_data.each do |user|
+
+
+    # Extract some fields
+    updated_at = user["updated_at"]
+    last_login_at = user["last_login_at"]
+    user_id = user["id"]
+    active = user["active"]
+    name = user["name"]
```

Comment:
`hard_delete` variable is not initialised. Can we add the following code to initialise the variable explicitly?

```suggestion
    name = user["name"]
    hard_delete = "false"
```

Reply:
Resolved as suggested.